frontend/src/stores/levels/actions.ts

```diff
@@ -1,7 +1,8 @@
-import { deleteRequest, post } from '@/services/http'
+import { deleteRequest, get, post, put } from '@/services/http'
+import { roles } from '../roles/state'
 import type { Level } from '@/domain/Level'
 import type { Role } from '@/domain/Role'
-import { roles } from '@/stores/roles/state'
+import { levels } from './state'
```

Comment:
You get your rebase wrongly. Levels state doesn't exist anymore, please, get back to the previous version